app/javascript/components/subject/CaseStatus.js

```diff
@@ -30,10 +31,11 @@ class CaseStatus extends React.Component {
   handleChange(event) {
     event.persist();
 
-    let value = event.target.type === 'checkbox' ? event.target.checked : event.target.value;
-    let hideModal = this.state.isolation && (value === 'Confirmed' || value === 'Probable');
+    const value = event.target.type === 'checkbox' ? event.target.checked : event.target.value;
+    const confirmedOrProbable = value === 'Confirmed' || value === 'Probable' ? true : false;
```

Comment:
You don't need a ternary for this, can change to:
`const confirmedOrProbable = value === 'Confirmed' || value === 'Probable'`